content/posts/how-to-use-playwright-in-cucumberjs/index.md

```diff
@@ -0,0 +1,157 @@
++++
+date = ""
+author = "nploi"
+description = "How to use Playwright in cucumberjs"
+title = "How to use Playwright in cucumberjs"
+categories = ["e2e test"]
```

Comment:
Can we get same categories with thiiss post https://github.com/manabie-com/manabie-com.github.io/blob/main/content/posts/why-we-use-cucumber-for-end-to-end-testing/index.md